hack/after-build/verify-generated-protobuf.sh

```diff
@@ -39,7 +39,7 @@ done
 # We would like to use "sudo" when running on Travis and
 # not use "sudo" when running on Jenkins.
 SUDO="sudo"
-sudo -h > /dev/null || SUDO=""
+sudo -h &> /dev/null || SUDO=""
```

Comment:
The comment makes it seem like this is only necessary on Travis. Is it still needed?